requirements-build.txt

```diff
@@ -81,10 +80,11 @@ wheel==0.43.0
     #   tzdata
 
 # The following packages are considered to be unsafe in a requirements file:
-setuptools==70.0.0 ; python_version >= "3.11" and python_version < "3.13"
+setuptools==69.0.2 ; python_version >= "3.11" and python_version < "3.13"
```

Comment:
Is this an intentional downgrade? Setup Tools `70.0.0` is the latest stable.

Reply:
requirements-build "downgrades" are not proper downgrades. these are all transitive dependencies, and are the best we could solve for our current deps.

This one is the result of upgrading ansible-runner, which has a specific setuptools requirement: https://github.com/ansible/ansible-runner/blob/2.4.0/pyproject.toml#L2

Reply:
requirements.txt (and poetry.lock) are the true source of our dependencies, which don't have a downgrade but rather an upgrade to the latest:
- https://github.com/quipucords/quipucords/pull/2676#discussion_r1629878846
- https://github.com/quipucords/quipucords/pull/2676#discussion_r1629880191